packages/editor/src/services/EditorHelperState.ts

```diff
@@ -66,5 +70,15 @@ export const EditorHelperState = defineState({
     'scaleSnap',
     'gridVisibility',
     'gridHeight'
-  ])
+  ]),
+  reactor: () => {
+    const [showGlbChildrenFlag] = useFeatureFlags([FeatureFlags.Studio.UI.Hierarchy.ShowGlbChildren])
+
+    useEffect(() => {
+      const showGlbChildren = getMutableState(EditorHelperState).showGlbChildren
+      showGlbChildren.set(showGlbChildrenFlag ?? true)
```

Comment:
rather than nullish coalesce, we should just not set the value of the useFeatureFlag hasn't actually fetched anything yet (if  `typeof showGlbChildrenFlag === 'undefined'` it seems)